RHDEVDOCS 6177 RN Pipelines 1.14.5 - undefined workspace info causes error now (#84146)
openshift/openshift-docs: modules/op-release-notes-1-14.adoc

* Before this update, when you used the web console and clicked a pipeline in the overview page, the pipeline details page did not contain information about tasks in the pipeline. With this update, when you click a pipeline in the overview page, the pipeline details page contains the required information.

* Before this update, when you configured {tekton-chains} to disable storing OCI artifacts by setting an empty `artifacts.oci.storage` value in the `TektonConfig` CR, the configuration did not work and {tekton-chains} attempted to store the artifacts and logged a failure in the `chains.tekton.dev/signed` annotation. With this update, when you set an empty `artifacts.oci.storage` value in the `TektonConfig` CR, {tekton-chains} does not attempt to store OCI artifacts.

> * Before this update, when you used workspace information, such as `$(workspaces.source.path)`, in a pipeline or task, but did not define a workspace, {pipelines-shortname} did not report an error. With this update, in this case {pipelines-shortname} reports an error and does not run the pipeline or task.

Reviewer: ```suggestion
* Previously when you used workspace information, such as `$(workspaces.source.path)`, in a pipeline or task without defining a workspace, {pipelines-shortname} did not report an error. {pipelines-shortname} now reports an error in this case and does not run the pipeline or task.
```

Author: Most of the styling is the boilerplate always used in this product, but I'm changing what I can.
